docs/source/development/reference/datatypes.rst

```diff
@@ -608,7 +608,8 @@ List structures
 
 .. c:type:: PJ_PRIME_MERIDIANS
 
-    Prime meridians defined in PROJ.
+    Hard-coded prime meridians. This is a legacy PROJ.4 feature,
```

Comment:
PROJ.4 by itself is legacy. Prime meridians aren't more legacy than +datum, +ellps or +units, are they?

Reply:
"legacy PROJ.4" wording removed.